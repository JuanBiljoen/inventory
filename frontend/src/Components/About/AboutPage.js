import { Card, Container, Stack } from "react-bootstrap";

//about page with dummy text for now
function AboutPage(props) {
  return (
    <Container style={{ marginTop: "15px" }}>
      <Stack gap={3}>
        <Card>
          <Card.Body>
            <Card.Title>
              <h3>About</h3>
            </Card.Title>
            <Card.Text>
              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce et
              placerat eros. Nullam sem eros, dapibus eu efficitur quis,
              tincidunt vitae lorem. Cras vitae viverra tellus. Pellentesque
              faucibus risus ac semper auctor. Aenean dictum mauris ornare
              vestibulum suscipit. Fusce sagittis leo scelerisque consectetur
              pellentesque. Integer nec ante maximus, viverra sapien eget,
              pulvinar urna. Suspendisse ut volutpat leo, vitae tempus est.
              Mauris ullamcorper, velit vel dapibus molestie, erat turpis
              sodales turpis, lacinia dignissim tellus nisi sit amet dui. Sed
              placerat ligula non orci porttitor vehicula.
            </Card.Text>
          </Card.Body>
        </Card>
        <Card>
          <Card.Body>
            <Card.Title>
              <h3>Location</h3>
            </Card.Title>
            <Card.Text>
              Vivamus a augue luctus, feugiat massa vitae, condimentum sapien.
              Nunc condimentum non sapien non efficitur. Pellentesque at orci a
              mi varius varius et non ex. Vivamus consectetur sem pellentesque
              nisi lobortis elementum. Donec et tellus at ligula suscipit
              lacinia at a mi. Curabitur ipsum felis, luctus eu bibendum nec,
              consectetur cursus ipsum. Donec tempor purus a ante mollis, eu
              rhoncus nisl tincidunt. Mauris sem mauris, porta vitae pharetra
              sed, iaculis eget felis. Aliquam vitae urna eu nunc finibus
              aliquet pretium in risus. Integer ligula metus, sagittis eget
              vulputate ac, hendrerit at purus. Fusce lectus sem, luctus
              placerat libero nec, dapibus accumsan dui. Nulla a luctus purus.
            </Card.Text>
          </Card.Body>
        </Card>
        <Card>
          <Card.Body>
            <Card.Title>
              <h3>Delivery</h3>
            </Card.Title>
            <Card.Text>
              Donec consectetur bibendum porttitor. Sed mi metus, porta eu diam
              a, blandit malesuada sem. Ut ut ex eu enim maximus tempor.
              Maecenas nec gravida nulla, vitae luctus enim. Aenean sit amet
              luctus massa. In sit amet tempus enim, ut egestas dolor. Curabitur
              maximus placerat mollis. Proin finibus ultricies nunc eu
              vestibulum.
            </Card.Text>
          </Card.Body>
        </Card>
        <Card>
          <Card.Body>
            <Card.Title>
              <h3>FAQ</h3>
            </Card.Title>
            <Card.Text>
              Cras non pulvinar erat, quis blandit sapien. Lorem ipsum dolor sit
              amet, consectetur adipiscing elit. Maecenas pretium porta dapibus.
              Sed imperdiet sollicitudin velit, in cursus nunc. Fusce sed lorem
              porta, aliquet ante sit amet, pulvinar ipsum. Nam augue quam,
              consequat id erat sed, mollis pulvinar risus. Morbi nulla turpis,
              gravida vel euismod nec, rhoncus ac mi. Duis sem dui, malesuada
              sit amet lectus eget, ornare suscipit mauris. Suspendisse potenti.
            </Card.Text>
          </Card.Body>
        </Card>
      </Stack>
    </Container>
  );
}

export default AboutPage;
