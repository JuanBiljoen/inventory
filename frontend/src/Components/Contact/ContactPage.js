import { useRef } from "react";
import { Button, Card, Container, Form, Stack } from "react-bootstrap";

// This page will allow the user enter their details that will be sent to the business 
function ContactPage(props) {
  // We use the useRef hooks below to help store the values the user inputs
  let nameRef = useRef("");
  let surnameRef = useRef("");
  let emailRef = useRef("");
  let phoneRef = useRef("");
  let additionalRef = useRef("");

 
  function submitContactInfo() {
    // Collect the user input values using the useRef hooks
    const name = nameRef.current.value;
    const surname = surnameRef.current.value;
    const email = emailRef.current.value;
    const phone = phoneRef.current.value;
    const additionalInfo = additionalRef.current.value;

    // Create a contact object with necesssary properties
    const contact = {
      name: name,
      surname: surname,
      email: email,
      phone: phone,
      additionalInfo: additionalInfo,
    };
   
    addContactHandler(contact);
  }

  // function for adding a user's details
  async function addContactHandler(contact) {
    const response = await fetch("/addcontact", {
      method: "POST",
      headers: {
        "Content-Type": "application/json",
      },
      body: JSON.stringify(contact),
    })
      .then((response) => {
        if (response.ok) {
          return response.json();
        }
        throw response;
      })
      .then((data) => {
        // The data received can only be true or false. Display an appropriate message to the user for either
        if (data) {
          alert("Your details were succesfully submitted");
        } else {
          alert("Something went wrong, please try again later.");
        }
      });
  }

  return (
    <Container style={{ marginTop: "15px" }} className="contact">
      <Card body className="mb-3">
        <h1>Contact Us</h1>
      </Card>
      <Stack
        direction="horizontal"
        className="mb-3"
        style={{ justifyContent: "center" }}
      >
        <Card className="me-3">
          <Card.Body>
            <Card.Title>Name</Card.Title>
            <Form.Group className="mb-3" controlId="formBasicName">
              <Form.Control type="text" placeholder="Your Name" ref={nameRef} />
            </Form.Group>
          </Card.Body>
        </Card>
        <Card className="me-3">
          <Card.Body>
            <Card.Title>Surname</Card.Title>
            <Form.Group className="mb-3" controlId="formBasicSurname">
              <Form.Control
                type="text"
                placeholder="Your Surname"
                ref={surnameRef}
              />
            </Form.Group>
          </Card.Body>
        </Card>
        <Card className="me-3">
          <Card.Body>
            <Card.Title>Email</Card.Title>
            <Form.Group className="mb-3" controlId="formBasicEmail">
              <Form.Control
                type="text"
                placeholder="Your Email"
                ref={emailRef}
              />
            </Form.Group>
          </Card.Body>
        </Card>
        <Card className="me-3">
          <Card.Body>
            <Card.Title>Phone</Card.Title>
            <Form.Group className="mb-3" controlId="formBasicPhone">
              <Form.Control
                type="text"
                placeholder="Your Phone Number"
                ref={phoneRef}
              />
            </Form.Group>
          </Card.Body>
        </Card>
      </Stack>

      <Card className="p-3">
        <Form>
          <Form.Group className="mb-3" controlId="formBasicAdditionalInfo">
            <Form.Label>Additional Info</Form.Label>
            <Form.Control
              as="textarea"
              rows={3}
              placeholder="Additonal Information"
              ref={additionalRef}
            />
          </Form.Group>
          <Button variant="primary" type="button" onClick={submitContactInfo}>
            Submit
          </Button>
        </Form>
      </Card>
    </Container>
  );
}

export default ContactPage;
