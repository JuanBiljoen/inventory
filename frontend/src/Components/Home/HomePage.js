import { Card, Container, Stack } from "react-bootstrap";
import { Facebook } from "react-bootstrap-icons";

//landing page 
function HomePage(props) {
  const truck1 = require("./../../resources/images/truck1.jpg");
  const truck2 = require("./../../resources/images/truck2.jpg");

  return (
    <Container style={{ marginTop: "15px" }} className="home">
      <Card body className="text-center">
        <h1 style={{ fontStyle: "italic", fontSize: "52px" }}>
          Welcome to SA Truck Parts
        </h1>
      </Card>
      <Stack direction="horizontal" className="mb-3 image-stack" gap={2}>
        <Stack direction="vertical">
          <Card.Img
            src={truck1}
            style={{ maxHeight: "400px" }}
          />
        </Stack>
        <Stack direction="vertical">
          <Card.Img
            src={truck2}
            style={{ maxHeight: "400px" }}
          />
        </Stack>
      </Stack>
      <Card>
        <Card.Body className="text-center">
          <Facebook
            className="me-1"
            style={{ fontSize: "32px", marginTop: "-12px" }}
          />
          <Card.Link
            style={{ fontSize: "32px" }}
            href="https://www.facebook.com/sacarandtruck"
            target="_blank"
          >
            Follow us on Facebook!
          </Card.Link>
        </Card.Body>
      </Card>
    </Container>
  );
}

export default HomePage;
