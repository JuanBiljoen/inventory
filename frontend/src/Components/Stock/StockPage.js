import { useEffect, useState } from "react";
import { Card, Container, Row, Stack } from "react-bootstrap";
import { getCookie } from "../../cookieService";
import AddProduct from "./Products/AddProduct";
import Product from "./Products/Product";
import AddRecommendedProduct from "./Recommended/AddRecommendedProduct";
import RecommendedProduct from "./Recommended/RecommendedProduct";

// This page is where users can view products for sale by the business
// Or they can recommend products to sell
function StockPage(props) {
  /**
   * @type {array}
   // productsData stores the product data fetched from an API call
   */
  const [productsData, setProductsData] = useState([]);
  /**
   * @type {array}
   * recommendedData stores the product recommendations fetched from an API call
   */
  const [recommendedData, setRecommendedData] = useState([]);
  /**
   * @type {boolean}
   *tokenExists indicates whether the user has a valid token or not
   */
  const [tokenExists, setTokenExists] = useState(false);
  /**
   * @type {string}
   * token stores the token cookie for the authenticated user
   */
  const [token, setToken] = useState(getCookie("token"));
  /**
   * @type {boolean}
   * isAdmin indicates whether the authenticated user is an admin or not
   */
  const [isAdmin, setIsAdmin] = useState(false);

  // UserId used to perform CRUD operations
  const userId = getCookie("user");

  // function to get cookies from admin and token and sets boolean values
  function checkCookie() {
    let isAdminCheck = getCookie("admin");
    setIsAdmin(isAdminCheck === "true");
    let tokenCheck = getCookie("token");
    setTokenExists(tokenCheck !== "");
  }

  // fetches data in db
  async function fetchProducts() {
    const response = await fetch("/allproducts", {
      method: "POST",
      headers: {
        "Content-Type": "application/json",
      },
    })
      .then((response) => {
        if (response.ok) {
          return response.json();
        }
        throw response;
      })
      .then((data) => {
        setProductsData(data);
      })
      .catch((error) => {
        console.error("Error fetching data: ", error);
      });
  }

  // fetch the Recommended Products in the API
  async function fetchRecommendedProducts() {
    const response = await fetch("/allrecommended", {
      method: "POST",
      headers: {
        "Content-Type": "application/json",
        Authorization: `Bearer ${token}`,
      },
      body: JSON.stringify({ userId: userId }),
    })
      .then((response) => {
        if (response.ok) {
          return response.json();
        }
        throw response;
      })
      .then((data) => {
        setRecommendedData(data);
      })
      .catch((error) => {
        console.error("Error fetching data: ", error);
      });
  }

  // adding product
  async function addProductHandler(product) {
    const response = await fetch("/addproduct", {
      method: "POST",
      headers: {
        "Content-Type": "application/json",
        Authorization: `Bearer ${token}`,
      },
      body: JSON.stringify(product),
    })
      .then((response) => {
        if (response.ok) {
          return response.json();
        }
        throw response;
      })
      .then((data) => {
        setProductsData(data);
        alert("Added information of a new product");
      });
  }

  // edit product
  async function editProductHandler(product) {
    const response = await fetch("/editproduct", {
      method: "PUT",
      headers: {
        "Content-Type": "application/json",
        Authorization: `Bearer ${token}`,
      },
      body: JSON.stringify(product),
    })
      .then((response) => {
        if (response.ok) {
          return response.json();
        }
        throw response;
      })
      .then((data) => {
        setProductsData(data);
      });
  }

  // deleting product
  async function deleteProductHandler(id) {
    const response = await fetch("/deleteproduct", {
      method: "DELETE",
      headers: {
        "Content-Type": "application/json",
        Authorization: `Bearer ${token}`,
      },
      body: JSON.stringify({ productId: id.value, userId: userId }),
    })
      .then((response) => {
        if (response.ok) {
          return response.json();
        }
        throw response;
      })
      .then((data) => {
        setProductsData(data);
      });
  }

  // adding product
  async function addRecommendationHandler(recommended) {
    const response = await fetch("/addrecommended", {
      method: "POST",
      headers: {
        "Content-Type": "application/json",
        Authorization: `Bearer ${token}`,
      },
      body: JSON.stringify(recommended),
    })
      .then((response) => {
        if (response.ok) {
          return response.json();
        }
        throw response;
      })
      .then((data) => {
        setRecommendedData(data);
      });
  }

  // editing added product
  async function editRecommendationHandler(recommended) {
    const response = await fetch("/editrecommended", {
      method: "PUT",
      headers: {
        "Content-Type": "application/json",
        Authorization: `Bearer ${token}`,
      },
      body: JSON.stringify(recommended),
    })
      .then((response) => {
        if (response.ok) {
          return response.json();
        }
        throw response;
      })
      .then((data) => {
        setRecommendedData(data);
      });
  }

  // deleting added product
  async function deleteRecommendationHandler(id) {
    const response = await fetch("/deleterecommended", {
      method: "DELETE",
      headers: {
        "Content-Type": "application/json",
        Authorization: `Bearer ${token}`,
      },
      body: JSON.stringify({ recommendedId: id.value, userId: userId }),
    })
      .then((response) => {
        if (response.ok) {
          return response.json();
        }
        throw response;
      })
      .then((data) => {
        setRecommendedData(data);
      });
  }

 //useEffect to fetch data
  useEffect(() => {
    fetchProducts();
    checkCookie(); // Check cookies on initial load
    if (tokenExists) {
      fetchRecommendedProducts();
    }
  }, []);

  return (
    <Container style={{ marginTop: "15px" }}>
      <Card body className="mb-3 menu-header header text-center">
        <h1 style={{ fontStyle: "italic" }}>
          Stock List
        </h1>
      </Card>

      {isAdmin && (
        <Row>
          <AddProduct onAddProduct={addProductHandler}></AddProduct>
        </Row>
      )}

      <hr />
      <Row>
        <Stack direction="horizontal" gap={2} className="products grid-stack">
          {productsData.map((product, index) => (
            <Product
              key={product._id}
              id={product._id}
              name={product.name}
              description={product.description}
              price={product.price}
              userId={userId}
              isAdmin={isAdmin}
              onEditProduct={editProductHandler}
              onDeleteProduct={deleteProductHandler}
            />
          ))}
        </Stack>
      </Row>

      <hr />
      {tokenExists && (
        <AddRecommendedProduct
          onAddRecommendedProduct={addRecommendationHandler}
        />
      )}

      <hr />
      <Row>
        <Stack
          direction="horizontal"
          gap={2}
          className="recommended grid-stack"
        >
          {recommendedData.map((recommended, index) => (
            <RecommendedProduct
              key={recommended._id}
              id={recommended._id}
              name={recommended.name}
              describe={recommended.describe}
              vin={recommended.vin}
              userId={userId}
              isAdmin={isAdmin}
              onEditRecommendedProduct={editRecommendationHandler}
              onDeleteRecommendedProduct={deleteRecommendationHandler}
            />
          ))}
        </Stack>
      </Row>
    </Container>
  );
}

export default StockPage;
