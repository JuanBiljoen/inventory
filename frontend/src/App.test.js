import renderer from "react-test-renderer";
import AboutPage from "./Components/About/AboutPage";
import ContactPage from "./Components/Contact/ContactPage";
import HomePage from './Components/Home/HomePage';
import StockPage from "./Components/Stock/StockPage";


test("snapshot test of HomePage", () => {
  const tree = renderer.create(<HomePage />).toJSON();
  expect(tree).toMatchSnapshot();
});

test("snapshot test of AboutPage", () => {
  const tree = renderer.create(<AboutPage />).toJSON();
  expect(tree).toMatchSnapshot();
});

test("snapshot test of MenuPage", () => {
  const tree = renderer.create(<StockPage />).toJSON();
  expect(tree).toMatchSnapshot();
});

test("snapshot test of ContactPage", () => {
  const tree = renderer.create(<ContactPage />).toJSON();
  expect(tree).toMatchSnapshot();
});