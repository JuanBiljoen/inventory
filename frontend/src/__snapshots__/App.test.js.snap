// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test of AboutPage 1`] = `
<div
  className="container"
  style={
    Object {
      "marginTop": "15px",
    }
  }
>
  <div
    className="vstack gap-3"
  >
    <div
      className="card"
    >
      <div
        className="card-body"
      >
        <div
          className="card-title h5"
        >
          <h3>
            About
          </h3>
        </div>
        <p
          className="card-text"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce et placerat eros. Nullam sem eros, dapibus eu efficitur quis, tincidunt vitae lorem. Cras vitae viverra tellus. Pellentesque faucibus risus ac semper auctor. Aenean dictum mauris ornare vestibulum suscipit. Fusce sagittis leo scelerisque consectetur pellentesque. Integer nec ante maximus, viverra sapien eget, pulvinar urna. Suspendisse ut volutpat leo, vitae tempus est. Mauris ullamcorper, velit vel dapibus molestie, erat turpis sodales turpis, lacinia dignissim tellus nisi sit amet dui. Sed placerat ligula non orci porttitor vehicula.
        </p>
      </div>
    </div>
    <div
      className="card"
    >
      <div
        className="card-body"
      >
        <div
          className="card-title h5"
        >
          <h3>
            Location
          </h3>
        </div>
        <p
          className="card-text"
        >
          Vivamus a augue luctus, feugiat massa vitae, condimentum sapien. Nunc condimentum non sapien non efficitur. Pellentesque at orci a mi varius varius et non ex. Vivamus consectetur sem pellentesque nisi lobortis elementum. Donec et tellus at ligula suscipit lacinia at a mi. Curabitur ipsum felis, luctus eu bibendum nec, consectetur cursus ipsum. Donec tempor purus a ante mollis, eu rhoncus nisl tincidunt. Mauris sem mauris, porta vitae pharetra sed, iaculis eget felis. Aliquam vitae urna eu nunc finibus aliquet pretium in risus. Integer ligula metus, sagittis eget vulputate ac, hendrerit at purus. Fusce lectus sem, luctus placerat libero nec, dapibus accumsan dui. Nulla a luctus purus.
        </p>
      </div>
    </div>
    <div
      className="card"
    >
      <div
        className="card-body"
      >
        <div
          className="card-title h5"
        >
          <h3>
            Delivery
          </h3>
        </div>
        <p
          className="card-text"
        >
          Donec consectetur bibendum porttitor. Sed mi metus, porta eu diam a, blandit malesuada sem. Ut ut ex eu enim maximus tempor. Maecenas nec gravida nulla, vitae luctus enim. Aenean sit amet luctus massa. In sit amet tempus enim, ut egestas dolor. Curabitur maximus placerat mollis. Proin finibus ultricies nunc eu vestibulum.
        </p>
      </div>
    </div>
    <div
      className="card"
    >
      <div
        className="card-body"
      >
        <div
          className="card-title h5"
        >
          <h3>
            FAQ
          </h3>
        </div>
        <p
          className="card-text"
        >
          Cras non pulvinar erat, quis blandit sapien. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas pretium porta dapibus. Sed imperdiet sollicitudin velit, in cursus nunc. Fusce sed lorem porta, aliquet ante sit amet, pulvinar ipsum. Nam augue quam, consequat id erat sed, mollis pulvinar risus. Morbi nulla turpis, gravida vel euismod nec, rhoncus ac mi. Duis sem dui, malesuada sit amet lectus eget, ornare suscipit mauris. Suspendisse potenti.
        </p>
      </div>
    </div>
  </div>
</div>
`;

exports[`snapshot test of ContactPage 1`] = `
<div
  className="contact container"
  style={
    Object {
      "marginTop": "15px",
    }
  }
>
  <div
    className="mb-3 card"
  >
    <div
      className="card-body"
    >
      <h1>
        Contact Us
      </h1>
    </div>
  </div>
  <div
    className="mb-3 hstack"
    style={
      Object {
        "justifyContent": "center",
      }
    }
  >
    <div
      className="me-3 card"
    >
      <div
        className="card-body"
      >
        <div
          className="card-title h5"
        >
          Name
        </div>
        <div
          className="mb-3"
        >
          <input
            className="form-control"
            id="formBasicName"
            placeholder="Your Name"
            type="text"
          />
        </div>
      </div>
    </div>
    <div
      className="me-3 card"
    >
      <div
        className="card-body"
      >
        <div
          className="card-title h5"
        >
          Surname
        </div>
        <div
          className="mb-3"
        >
          <input
            className="form-control"
            id="formBasicSurname"
            placeholder="Your Surname"
            type="text"
          />
        </div>
      </div>
    </div>
    <div
      className="me-3 card"
    >
      <div
        className="card-body"
      >
        <div
          className="card-title h5"
        >
          Email
        </div>
        <div
          className="mb-3"
        >
          <input
            className="form-control"
            id="formBasicEmail"
            placeholder="Your Email"
            type="text"
          />
        </div>
      </div>
    </div>
    <div
      className="me-3 card"
    >
      <div
        className="card-body"
      >
        <div
          className="card-title h5"
        >
          Phone
        </div>
        <div
          className="mb-3"
        >
          <input
            className="form-control"
            id="formBasicPhone"
            placeholder="Your Phone Number"
            type="text"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="p-3 card"
  >
    <form
      className=""
    >
      <div
        className="mb-3"
      >
        <label
          className="form-label"
          htmlFor="formBasicAdditionalInfo"
        >
          Additional Info
        </label>
        <textarea
          className="form-control"
          id="formBasicAdditionalInfo"
          placeholder="Additonal Information"
          rows={3}
        />
      </div>
      <button
        className="btn btn-primary"
        disabled={false}
        onClick={[Function]}
        type="button"
      >
        Submit
      </button>
    </form>
  </div>
</div>
`;

exports[`snapshot test of HomePage 1`] = `
<div
  className="home container"
  style={
    Object {
      "marginTop": "15px",
    }
  }
>
  <div
    className="text-center card"
  >
    <div
      className="card-body"
    >
      <h1
        style={
          Object {
            "fontSize": "52px",
            "fontStyle": "italic",
          }
        }
      >
        Welcome to SA Truck Parts
      </h1>
    </div>
  </div>
  <div
    className="mb-3 image-stack hstack gap-2"
  >
    <div
      className="vstack"
    >
      <img
        className="card-img"
        src="truck1.jpg"
        style={
          Object {
            "maxHeight": "400px",
          }
        }
      />
    </div>
    <div
      className="vstack"
    >
      <img
        className="card-img"
        src="truck2.jpg"
        style={
          Object {
            "maxHeight": "400px",
          }
        }
      />
    </div>
  </div>
  <div
    className="card"
  >
    <div
      className="text-center card-body"
    >
      <svg
        className="me-1"
        fill="currentColor"
        height="1em"
        style={
          Object {
            "fontSize": "32px",
            "marginTop": "-12px",
          }
        }
        viewBox="0 0 16 16"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M16 8.049c0-4.446-3.582-8.05-8-8.05C3.58 0-.002 3.603-.002 8.05c0 4.017 2.926 7.347 6.75 7.951v-5.625h-2.03V8.05H6.75V6.275c0-2.017 1.195-3.131 3.022-3.131.876 0 1.791.157 1.791.157v1.98h-1.009c-.993 0-1.303.621-1.303 1.258v1.51h2.218l-.354 2.326H9.25V16c3.824-.604 6.75-3.934 6.75-7.951z"
        />
      </svg>
      <a
        className="card-link"
        href="https://www.facebook.com/sacarandtruck"
        style={
          Object {
            "fontSize": "32px",
          }
        }
        target="_blank"
      >
        Follow us on Facebook!
      </a>
    </div>
  </div>
</div>
`;

exports[`snapshot test of MenuPage 1`] = `
<div
  className="container"
  style={
    Object {
      "marginTop": "15px",
    }
  }
>
  <div
    className="mb-3 menu-header header text-center card"
  >
    <div
      className="card-body"
    >
      <h1
        style={
          Object {
            "fontStyle": "italic",
          }
        }
      >
        Stock List
      </h1>
    </div>
  </div>
  <hr />
  <div
    className="row"
  >
    <div
      className="products grid-stack hstack gap-2"
    />
  </div>
  <hr />
  <hr />
  <div
    className="row"
  >
    <div
      className="recommended grid-stack hstack gap-2"
    />
  </div>
</div>
`;
